templates/zerver/app/left_sidebar.html

```diff
@@ -48,6 +50,7 @@
                 </a>
                 <span class="arrow starred-messages-sidebar-menu-icon"><i class="zulip-icon zulip-icon-ellipsis-v-solid" aria-hidden="true"></i></span>
             </li>
+            {% endif %}
```

Comment:
What do you think about having these elements be still present in the DOM, and just have a `hidden_if_web_public` class on the element, which does `display: none !important;`?  

I'm thinking that strategy could be better from the standpoint of avoiding problems if we have JavaScript code that expects these elements to exist in the page.

Reply:
Done. I just left one in `actions_popover_content.hbs` since that is a simpler change that adding a class to all the list items.